tests/framework/jailer.py

```diff
@@ -256,7 +256,13 @@ def cleanup(self):
             shutil.rmtree(self.chroot_base_with_id(), ignore_errors=True)
 
         if self.netns and os.path.exists("/var/run/netns/{}".format(self.netns)):
-            utils.run_cmd("ip netns del {}".format(self.netns))
+            try:
+                utils.run_cmd("ip netns del {}".format(self.netns))
+            except ChildProcessError:
+                # Sometimes, a race condition in pytest causes this destructor to run twice.
```

Comment:
can we add a #TODO so that we don't forget this?

Reply:
Since @pb8o already has a fix in-flight for this, I this would send the wrong sign to the community imo